Cargo.toml

```diff
@@ -34,7 +35,7 @@ read_core = []
 # Read support for all file formats (including unaligned files).
 read = ["read_core", "archive", "coff", "elf", "macho", "pe", "unaligned"]
 # Core write support. You will need to enable some file formats too.
-write_core = ["crc32fast", "indexmap/std", "std"]
+write_core = ["crc32fast", "indexmap", "hashbrown"]
```

Comment:
This extra dependency is unnecessary when libstd is available.

Reply:
Thanks, you're correct there. Since cargo doesn't support negating features AFAIK, is there a clean way to represent only including the `hashmap` dependency if `std` is not enabled? Apologies, I'm not a cargo expert and don't know the best practices here.

Reply:
I am not sure there is a way at all unfortunately.

Reply:
I think this is ok if `indexmap` depends on it anyway.

Reply:
Sounds good, I'll leave it as is considering there is no real alternative.